### DISCUSSION TITLE
SOC Auth log

### DISCUSSION THREAD

**Discussion by Sahale** [Security-Onion-Solutions/securityonion#10127] [Unsupported Versions] [answered]
Hello!
Earlier it was possible to get username of logged in users with this instruction: [SOC Auth](https://docs.securityonion.net/en/2.3/soc-logs.html), but now there is no sqlite DB. How can I find out who logged in via web interface?
Thanks!

**Comment by InfosecGoon** [comment, marked as answer]
The database has been moved to /nsm/kratos/db/db.sqlite .
